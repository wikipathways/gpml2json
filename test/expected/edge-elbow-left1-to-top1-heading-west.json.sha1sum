8cc84581b8fd7ca9696deeca204aa47d620b0d78 ?-
