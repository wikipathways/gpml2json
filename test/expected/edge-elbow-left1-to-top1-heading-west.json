{"pathway":{"@context":["https://cdn.rawgit.com/wikipathways/WpVocabularies/7a46a05/contexts/pvjs.jsonld"],"backgroundColor":"white","borderWidth":0,"color":"black","contains":["ebee3","e691a","a00ff"],"drawAs":"rect","gpmlElementName":"Pathway","height":818.1126371280204,"kaavioType":"Group","name":"test case","padding":5,"type":["Pathway"],"width":1047.7865523249097,"x":0,"y":0,"zIndex":0,"fontSize":12,"fontWeight":"bold","textAlign":"left","verticalAlign":"top","dataSourceVersion":"20130813","standardName":"test case","displayName":"test case","textContent":"Name: test case","isSimilarTo":"http://wikipathways.org/index.php/Special:SearchPathways?query=test%20case&doSearch=1","id":"hash://sha256/84ca28313988fc495392d60f8035e3ba016250c7dac5cc47ab7f8741b76b8e9d"},"entityMap":{"e691a":{"id":"e691a","textContent":"0017","wpType":"GeneProduct","x":22.01847189693042,"y":81.49192448464771,"fontSize":10,"height":20,"verticalAlign":"middle","width":80,"zIndex":32960,"textAlign":"center","color":"#000000","backgroundColor":"#ffffff","textDecoration":"none","fontFamily":"Arial","fontStyle":"normal","fontWeight":"normal","borderWidth":1,"drawAs":"rect","lineHeight":1.1,"overflow":"hidden","textOverflow":"clip","whiteSpace":"pre","padding":8,"type":["DataNode","SingleFreeNode","GeneProduct"],"kaavioType":"SingleFreeNode","gpmlElementName":"DataNode"},"a00ff":{"id":"a00ff","textContent":"GeneProduct","wpType":"GeneProduct","x":194.0067284219814,"y":132.68506052813458,"fontSize":10,"height":20,"verticalAlign":"middle","width":80,"zIndex":32961,"textAlign":"center","color":"#000000","backgroundColor":"#ffffff","textDecoration":"none","fontFamily":"Arial","fontStyle":"normal","fontWeight":"normal","borderWidth":1,"drawAs":"rect","lineHeight":1.1,"overflow":"hidden","textOverflow":"clip","whiteSpace":"pre","padding":8,"type":["DataNode","SingleFreeNode","GeneProduct"],"kaavioType":"SingleFreeNode","gpmlElementName":"DataNode"},"ebee3":{"id":"ebee3","drawAs":"ElbowLine","borderWidth":1,"zIndex":32959,"points":[{"isAttachedTo":"a00ff","attachmentDisplay":{"position":[0,0.5],"offset":[0,0]},"x":194.0067284219814,"y":142.68506052813458,"orientation":[-1,0]},{"x":174.0067284219814,"y":102.08849250639115},{"y":61.49192448464771,"x":118.0126001594559},{"isAttachedTo":"e691a","attachmentDisplay":{"position":[0.5,0],"offset":[0,0]},"x":62.01847189693042,"y":81.49192448464771,"orientation":[0,1]}],"color":"#000000","type":["Interaction","Edge"],"kaavioType":"Edge","gpmlElementName":"Interaction","markerStart":"TBar","markerEnd":"Arrow","isAttachedTo":["a00ff","e691a"]}}}
